datacollectiondemo/source/cpp/build.bat

```diff
@@ -76,13 +76,14 @@ goto help
         md %BUILD_DIR%
         cd %BUILD_DIR%
         cmake -G "NMake Makefiles" ^
-	      -DKAA_DEBUG_ENABLED=1 ^
+              -DCMAKE_BUILD_TYPE=Debug ^
               -DKAA_WITHOUT_EVENTS=1 ^
-              -DKAA_WITHOUT_CONFIGURATION=1 ^
+              -DKAA_WITHOUT_LOGGING=1 ^
```

Comment:
Should we disable Logging in the Data collection demo? AFAIK this is a key feature here. Am I wrong?